package/Dockerfile

```diff
@@ -36,35 +25,82 @@ RUN git clone https://github.com/longhorn/go-spdk-helper.git ${GO_SPDK_HELPER_DI
     rm -rf ${GO_SPDK_HELPER_DIR}
 
 # Build liblonghorn
+ENV LIBLONGHORN_COMMIT_ID 53d1c063b95efc8d949b095bd4bf04637230265f
 RUN cd /usr/src && \
     git clone https://github.com/rancher/liblonghorn.git && \
     cd liblonghorn && \
-    git checkout 53d1c063b95efc8d949b095bd4bf04637230265f && \
+    git checkout ${LIBLONGHORN_COMMIT_ID} && \
     make; \
     make install
 
 # Build TGT
+ENV TGT_COMMIT_ID 3a8bc4823b5390e046f7aa8231ed262c0365c42c
 RUN cd /usr/src && \
     git clone https://github.com/rancher/tgt.git && \
     cd tgt && \
-    git checkout 3a8bc4823b5390e046f7aa8231ed262c0365c42c && \
+    git checkout ${TGT_COMMIT_ID} && \
     make; \
     make install
 
 # Build SPDK
 ENV SPDK_DIR /usr/src/spdk
-RUN if [ ${ARCH} == "amd64" ]; then \
-        git clone https://github.com/longhorn/spdk.git ${SPDK_DIR} --recursive && \
+ENV SPDK_COMMIT_ID 980f535d3876a5a6665486ad0cfaac5d584034aa
+RUN git clone https://github.com/longhorn/spdk.git ${SPDK_DIR} --recursive && \
+    if [ ${ARCH} == "amd64" ]; then \
         cd ${SPDK_DIR} && \
-        git checkout 980f535d3876a5a6665486ad0cfaac5d584034aa && \
+        git checkout ${SPDK_COMMIT_ID} && \
         git submodule update --init && \
         ./scripts/pkgdep.sh && \
         ./configure --target-arch=nehalem --disable-tests --disable-unit-tests --disable-examples && \
         make -j$(nproc) && \
-        make install && \
-        rm -rf ${SPDK_DIR}; \
+        make install; \
     fi
 
+FROM registry.suse.com/bci/bci-base:15.4 AS release
+
+ARG ARCH=amd64
+
+RUN zypper -n addrepo --refresh https://download.opensuse.org/repositories/system:/snappy/SLE_15/system:snappy.repo && \
+    zypper -n addrepo --refresh https://download.opensuse.org/repositories/network:/utilities/SLE_15_SP4/network:utilities.repo && \
+    zypper -n addrepo --refresh https://download.opensuse.org/repositories/devel:libraries:c_c++/15.4/devel:libraries:c_c++.repo && \
+    zypper -n addrepo --refresh https://download.opensuse.org/repositories/devel:languages:python:Factory/15.4/devel:languages:python:Factory.repo && \
+    zypper -n addrepo --refresh https://download.opensuse.org/repositories/devel:languages:python:backports/SLE_15/devel:languages:python:backports.repo && \
+    zypper --gpg-auto-import-keys ref
+
+RUN zypper -n install kmod curl fuse wget \
+    nfs-client nfs4-acl-tools cifs-utils \
+    libibverbs librdmacm1 rdma-core-devel perl-Config-General libaio-devel sg3_utils \
+    iputils telnet iproute2 qemu-tools \
+    e2fsprogs e2fsprogs-devel xfsprogs xfsprogs-devel util-linux-systemd python3
+
+# Install SPDK dependencies
+COPY --from=builder /usr/src/spdk/scripts /tmp/spdk
```

Comment:
So `/tmp/spdk/pkgdep.sh` depends on `/tmp/spdk`? What's content in scripts folder?

Reply:
`/tmp/spdk` contains the scripts and utilities for 
- dependencies installation
- rpc python files
- performance measurement
- ...